docs/Using Fleet/enroll-hosts.md

```diff
@@ -1,316 +1,93 @@
-
 # Enroll hosts
 
-- [Enroll hosts](#enroll-hosts)
-  - [Introduction](#introduction)
-  - [Supported osquery versions](#supported-osquery-versions)
-  - [Add hosts with Fleetd](#add-hosts-with-fleetd)
-
-    - [Signing installers](#signing-installers)
-    - [Including Fleet Desktop](#including-fleet-desktop)
-    - [Adding multiple hosts](#adding-multiple-hosts)
-    - [Automatically adding hosts to a team](#automatically-adding-hosts-to-a-team)
-    - [Generating Windows installers using local WiX toolset](#generating-windows-installers-using-local-wix-toolset)
-    - [Configuration options](#configuration-options)
-  - [Add hosts with plain osquery](#add-hosts-with-plain-osquery)
-    - [Set up your Fleet enroll secret](#set-up-your-fleet-enroll-secret)
-    - [Provide the TLS certificate that osquery will use to communicate with Fleet](#provide-the-tls-certificate-that-osquery-will-use-to-communicate-with-fleet)
-    - [Configure and launch osquery](#configure-and-launch-osquery)
-      - [Launching osqueryd using command-line flags](#launching-osqueryd-using-command-line-flags)
-      - [Launching osqueryd using a flag file](#launching-osqueryd-using-a-flag-file)
-    - [Migrating from plain osquery to osquery installer](#migrating-from-plain-osquery-to-osquery-installer)
-      - [Generate installer](#generate-installer)
-      - [Migrate](#migrate)
-  - [Add Chromebooks with the fleetd Chrome extension](#add-chromebooks-with-the-fleetd-chrome-extension)
-  - [Grant full disk access to osquery on macOS](#grant-full-disk-access-to-osquery-on-macos)
-    - [Creating the configuration profile](#creating-the-configuration-profile)
-      - [Obtaining identifiers](#obtaining-identifiers)
-      - [Creating the profile](#creating-the-profile)
-      - [Test the profile](#test-the-profile)
-
 ## Introduction
 
-Fleet gathers information from an [osquery](https://github.com/osquery/osquery) agent installed on each of your hosts. The recommended way to install osquery is using Fleetd, which includes [Orbit](https://fleetdm.com/docs/using-fleet/fleetd), Fleet's lightweight osquery runtime and auto-updater.
+Fleet gathers information from an [osquery](https://github.com/osquery/osquery) agent installed on each of your hosts. The recommended way to install osquery is using fleetd.
```

Comment:
@rachaelshaw I'll just leave one comment for all suggestions here. Looks great to me.

I like how you cut "How to enroll?" section since we already have H1 -> Enroll hosts at the top. I like the idea of having UI, CLI, and other sections as H2, so they appear on the side nav. Cutting instructions for team based fleet packages is great.

If you can commit all changes it would be great, so I can run it locally and take one more look. I can take care of broken links after you make changes. Thank you for the extensive review! 🙌

Reply:
@marko-lisica done!